lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -264,7 +264,21 @@ def add_copy_resources_script_phase
             output_paths = []
             unless resource_paths_by_config.values.all?(&:empty?)
               input_paths = [target.copy_resources_script_relative_path, *resource_paths_by_config.values.flatten.uniq]
-              output_paths = ['${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}']
+
+              output_paths = resource_paths_by_config.values.flatten.uniq.map do |input_path| 
```

Comment:
extract `resource_paths_by_config.values.flatten.uniq` into a variable so it can be used for `input_paths` above as well.